compiler/luci/service/src/Nodes/CircleDepthwiseConv2D.cpp

```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
  */
 
+#include "HelperConv2Ds.h"
```

Comment:
(optional) 

How about renaming a header file to `Conv2DHelepr.h` 
I found out that there are more headers named `*.Helper.h` than `Helper*.h`. 
So I thought the name `*Helper.h` might be good for consistency. 

```bash 
ONE/compiler [$?] …
➜ find -name '*Helper.h' | wc -l
36

ONE/compiler [$?] …
➜ find -name 'Helper*.h' | wc -l 
10
```

-----------------

But still, it is a personal preference, So I don't want to force it at all. 
Please feel free to make your own decision.

Reply:
(optional 2)

It would be also good to move the `infer_conv2d_type` helper function to the `CircleShapeInferenceHelper.h` header file directly, for the reusability of helper functions.

As I know, because it is a template function, it is okay that its definition can be located in the header file.

Reply:
> As I know, because it is a template function, it is okay that its definition can be located in the header file. 

Ah, there was a history about `HelperPads` introduced. If so, your suggestion looks good for me :)